doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -220,63 +226,85 @@ Requests to the environment
    .. seealso:: Section :ref:`performingcomputations`.
 
 
-.. cmd:: Print Assumptions @qualid
+.. cmd:: Print Assumptions @smart_qualid
 
-   This commands display all the assumptions (axioms, parameters and
+   Displays all the assumptions (axioms, parameters and
    variables) a theorem or definition depends on. Especially, it informs
    on the assumptions with respect to which the validity of a theorem
    relies.
```

Comment:
Indeed, this second sentence is redundant. What would be useful OTOH is to show the message printed by the command when it depends on no assumption at all ("Closed under the global context").